specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-05-01/containerregistry.json

```diff
@@ -1617,7 +1620,8 @@
       "properties": {
         "days": {
           "format": "int32",
-          "description": "The number of days to retain manifest before it expires.",
+          "description": "The number of days to retain an untagged manifest after which it gets purged.",
+          "default": 7,
```

Comment:
what it is for existing user if not setting this property?

Reply:
It was always defaulted to 7 in server side. Just not exposed in swagger.